src/index.py

```diff
@@ -0,0 +1,47 @@
+import os
+from dotenv import load_dotenv
+import requests
+import json
+# import pandas as pd
+# from pandas import json_normalize
+
+load_dotenv()
+
+api_key = os.getenv("API_KEY")
+userkey = {"user_key":api_key}
+
+query = {
+  "field_ids": [
+    "name",
+    "short_description",
+    "website_url",
+    "linkedin",
+  ],
+  "query": [
+    {
+      "type": "predicate",
+      "field_id": "location_identifiers",
+      "operator_id": "includes",
+      "values": [
+        "Europe"
+      ]
+    }
+  ],
+  "limit": 10
+}
+
+def company_count(query):
```

Comment:
I think this function name should be more descriptive to be something like `get_companies(query_params):`, and you should better handle errors using try and catch functions